deckhouse-controller/entrypoint.sh

```diff
@@ -34,4 +34,37 @@ EOF
 
 cat ${MODULES_DIR}/values-${bundles_map[$bundle]}.yaml >> ${MODULES_DIR}/values.yaml
 
-exec /sbin/tini -- /usr/bin/deckhouse-controller start
+set +o pipefail
+set +e
+
+PID=0
+EXITCODE=0
+
+for SIG in SIGUSR1 SIGUSR2 SIGINT SIGTERM SIGHUP SIGQUIT; do
+  trap "signal_handler ${SIG}" "${SIG}"
+done
```

Comment:
Let's move the logic under the functions definitions

Reply:
Could you be a little more specific?

Reply:
solved through videocall